test/util/tools/SceApiTester.js

```diff
@@ -0,0 +1,57 @@
+process.env.NODE_ENV = 'test';
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+chai.should();
+chai.use(chaiHttp);
+
+class SceApiTester{
+  constructor(app){
+    this.app = app;
+  }
+
+  async sendPostRequest(endpoint, params){
+    let response = null;
+    await chai
+      .request(this.app)
+      .post(endpoint)
+      .send(params)
+      .then(function(res) {
+        response = res;
+      })
+      .catch(err =>{
+        throw err;
+      });
+    return response;
+  }
+  async sendPostRequestWithToken(token, endpoint, params){
```

Comment:
add space between this function and end of the one above
```suggestion

  async sendPostRequestWithToken(token, endpoint, params){
```